.gitignore

```diff
@@ -7,8 +7,8 @@
 .project
 .settings
 bin
-build
 target
+build
```

Comment:
This is really minor, but not so obvious change.
Therefore it is really would be great to have a CLA signed from you.
However it might not be needed at all since this change looks like not reasonable.
What is the difference between previous content of this file and what you have changed?
Looks like we still ignore that `build` dir.
So, why do we need your change?

Thanks for understanding